datadog_cluster_agent/metadata.csv

```diff
@@ -34,6 +34,7 @@ datadog.cluster_agent.cluster_checks.updating_stats_duration_seconds,gauge,,seco
 datadog.cluster_agent.datadog.rate_limit_queries.limit,gauge,,query,,Maximum number of queries to the Datadog API allowed in the period by endpoint,0,datadog_cluster_agent,rate limit queries limit,
 datadog.cluster_agent.datadog.rate_limit_queries.period,gauge,,second,,Period of rate limiting for the Datadog API by endpoint,0,datadog_cluster_agent,rate limit queries period,
 datadog.cluster_agent.datadog.rate_limit_queries.remaining,gauge,,query,,Number of queries to the Datadog API remaining before next reset by endpoint,0,datadog_cluster_agent,rate limit queries remaining,
+datadog.cluster_agent.datadog.rate_limit_queries.remaining_min,gauge,,query,,Minimum number of queries remaining before next reset observed during an expiration interval of 2*refresh period,0,rate limit queries remaining minimum,
```

Comment:
i think the integration name may be missing here
```suggestion
datadog.cluster_agent.datadog.rate_limit_queries.remaining_min,gauge,,query,,Minimum number of queries remaining before next reset observed during an expiration interval of 2*refresh period,0,datadog_cluster_agent,rate limit queries remaining minimum,
```

Reply:
Thanks @jennchenn for spotting this.